docs/rules-engine/operators.md

```diff
@@ -22,7 +22,53 @@ Example of usage :
 }
 ```
 
-Full operator list provided by default: [Operators](https://dev.azure.com/AmadeusDigitalAirline/DES%20Platform/_wiki/wikis/DES%20Documentation/1954/Content-rules?anchor=operator-list)  
+## Full operator list provided by default
```

Comment:
I think we are missing the operator key, which is the most important part if this is used out of the CMS context